app/theme/client/imports/general/base.css

```diff
@@ -239,7 +239,7 @@ button {
 	.rc-old .main-content-flex {
 		flex: 1 0 auto !important; /* 1 */
 
-		height: auto !important; /* 1 */ 
+		height: auto !important; /* 1 */
```

Comment:
Actually, no 😬 
But since only trailing spaces are being removed, I think we can keep these changes, can't we? (or is it better to put them back there?)

Reply:
Well, I noticed the same change appeared on my local 😬 so probably it's better to merge it with this anyways